### ISSUE TITLE
[trasferimenti] Adeguamento - Revisione 0 del 23 settembre 2016

### ISSUE THREAD

**Issue by PaoloGiustiniani** [CroceRossaItaliana/jorvik#399]
TRASFERIMENTI

9.1 Il Volontario C.R.I. che intende trasferirsi ad altro Comitato C.R.I., presenta motivata richiesta al proprio Presidente **(Già presente in GAIA)**

9.2 Il trasferimento è disposto, senza altra formalità, di concerto fra i due Presidenti di Comitato C.R.I. In tal caso il Presidente del Comitato C.R.I. di provenienza trasmette il fascicolo, nel rispetto della normativa in materia di riservatezza dei dati personali. Del trasferimento è data idonea informativa al competente Presidente Regionale. **(Bisogna implementare la notifica al Presidente Regionale)**

9.3 La richiesta di trasferimento non può essere rigettata, salvo che la motivazione sia assente o incongrua. Il rigetto è impugnabile dinanzi al Presidente Regionale. **(@luca-dex come la interpreti?)**

9.4 La richiesta di trasferimento può essere sospesa con provvedimento motivato esclusivamente nel caso in cui il Volontario C.R.I. sia soggetto a procedimento disciplinare o nel caso in cui siano indette le elezioni per il rinnovo degli organi statutari del Comitato o dei Comitati interessati. In tal caso, il trasferimento è
disposto non appena il procedimento disciplinare sia stato definito o la procedura elettorale espletata. **(Oltre alla funzionalità di accetta o rifiuta dobbiamo inserire un sospendi @luca-dex ?)**

9.5 In mancanza di decisione espressa, trascorsi inutilmente trenta giorni dalla richiesta di trasferimento, esso si intende comunque autorizzato. **(Dobbiamo verificare il corretto funzionamento delle auto approvazioni, in passato non funzionavano)**

**Comment by luca-dex** [comment]
> 9.2  Il trasferimento è disposto, senza altra formalità, di concerto fra i due Presidenti di Comitato C.R.I. [...] 

@AlfioEmanueleFresta @matthewmazzoleni mi confermate che al momento tutti e due i presidenti devono dare l'ok al trasferimento prima che questo sia effettivamente disposto?

> 9.3 La richiesta di trasferimento non può essere rigettata, salvo che la motivazione sia assente o incongrua. Il rigetto è impugnabile dinanzi al Presidente Regionale. (@luca-dex come la interpreti?)

Notificherei al regionale, ai due presidenti e alla persona interessata il rigetto (che deve includere la motivazione). Nella stessa comunicazione citerei questo punto del regolamento, dando le indicazioni per presentare al Presidente Regionale il ricorso. Onestamente non gestirei questa cosa tramite gaia, se la gestiscono a parte e in base alla decisione del Presidente Regionale si trasferisce la persona. Questo potrebbe essere gestito in due modi:
* nuovo trasferimento
* il presidente regionale può aprire la vecchia richiesta e ha un bel tasto per effettuare il trasferimento comunque, indicando il riferimento dell'autorizzazione concessa a seguito di ricorso 

> 9.4 La richiesta di trasferimento può essere sospesa con provvedimento motivato esclusivamente nel caso in cui il Volontario C.R.I. sia soggetto a procedimento disciplinare o nel caso in cui siano indette le elezioni per il rinnovo degli organi statutari del Comitato o dei Comitati interessati. In tal caso, il trasferimento è
disposto non appena il procedimento disciplinare sia stato definito o la procedura elettorale espletata. (Oltre alla funzionalità di accetta o rifiuta dobbiamo inserire un sospendi @luca-dex ?)

Lo vedo un po' come uno stato della richiesta di trasferimento che blocca il conteggio dei giorni dell'approvazione automatica. La mia domanda è: chi effettua la sospensione? Possibili risposte che mi vengono in mente:
* il presidente del comitato di provenienza (per la parte relativa al provvedimento disciplinare)
* l'organo CRI che indice le elezioni in un comitato, quando uno dei comitati va ad elezioni, anche se non mi è chiaro come questa cosa dovrebbe accadere, sopratutto perchè vorrebbe dire avere una serie di meccanismi da mettere in atto quando vengono indette le elezioni

> 9.5 In mancanza di decisione espressa, trascorsi inutilmente trenta giorni dalla richiesta di trasferimento, esso si intende comunque autorizzato. (Dobbiamo verificare il corretto funzionamento delle auto approvazioni, in passato non funzionavano)

Credo che  basti aspettare che #315 venga mergiata

**Comment by AlfioEmanueleFresta** [comment]
> @AlfioEmanueleFresta @matthewmazzoleni mi confermate che al momento tutti e due i presidenti devono dare l'ok al trasferimento prima che questo sia effettivamente disposto?

@luca-dex No, credo che la giustificazione sia che il nuovo presidente non puo' rifiutare il trasferimento in ingresso? @matthewmazzoleni

**Comment by luca-dex** [comment]
Ok, penso che dipenda dal significato di *"di concerto"*. Almeno notificahiamo il presidente ricevente dell'avvio dell'iter di estensione / trasferimento di un volontario verso il suo comitato? O si trova in casa il nuovo volontario direttamente

**Comment by PaoloGiustiniani** [comment]
Chiederei anche qui conferma a @galamarco

**Comment by galamarco** [comment]
> Ok, penso che dipenda dal significato di "di concerto". Almeno notificahiamo il presidente ricevente dell'avvio dell'iter di estensione / trasferimento di un volontario verso il suo comitato? O si trova in casa il nuovo volontario direttamente

Concordo con @luca-dex , deve essere notificato in qualche modo chi entra negli elenchi soci del Comitato (io lo notificherei non solo al Presidente ma anche all'ufficio soci per concordare un briefing iniziale con il nuovo arrivato)

**Comment by AlfioEmanueleFresta** [comment]
Attualmente notifichiamo gia' il presidente della Sede di destinazione. Questo avviene immediatamente alla richiesta di trasferimento, e gli viene inoltrata una copia della richiesta in formato PDF. Credo che la motivazione dietro la notifica al momento della richiesta, piuttosto che al momento dell'approvazione da parte del Presidente del Comitato uscente, sia dovuto alla interpretazione data al 'di concerto'.

**Comment by PaoloGiustiniani** [comment]
Open check-list (vi invito a modificarla)

- [ ] Notificare il trasferimento (inizio e fine) al presidente regionale, presidente territoriale ed ufficio soci di competenza

- [ ] Notificherei al regionale, ai due presidenti e alla persona interessata il rigetto (che deve includere la motivazione). Nella stessa comunicazione citerei questo punto del regolamento, dando le indicazioni per presentare al Presidente Regionale il ricorso. Onestamente non gestirei questa cosa tramite gaia, se la gestiscono a parte e in base alla decisione del Presidente Regionale si trasferisce la persona. Questo potrebbe essere gestito in due modi:

nuovo trasferimento
il presidente regionale può aprire la vecchia richiesta e ha un bel tasto per effettuare il trasferimento comunque, indicando il riferimento dell'autorizzazione concessa a seguito di ricorso

**Comment by luca-dex** [comment]
Ma notificare il presidente regionale non può semplicemente voler dire che lui ha a disposizione il pannello dal quale vede i trasferimenti in atto? In caso di trasferimenti extra regione i presidenti regionali interessati sono 2

**Comment by AlfioEmanueleFresta** [comment]
Concordo con @luca-dex, considerando che il Regionale non deve effettuare alcuna azione, e che la notifica si 'perderebbe' poco dopo, credo la cosa piu' opportuna sia un pannello. Se vogliamo farla bene, in termini tecnici costa molto poco fare un pannello dove ogni Presidente Regionale possa vedere la lista di tutti i trasferimenti avvenuti in regione, in ordine dal piu' recente, magari con indicazione di stato (in corso, negato, approvato). Questo manterrebbe pure lo storico (il Presidente puo' vedere e ricercare l'elenco anche per i trasferimenti passati). Che ve ne pare?

**Comment by luca-dex** [comment]
@AlfioEmanueleFresta mi leggi nel pensiero

**Comment by luca-dex** [comment]
Già che c'ero ho provato a richiedere un trasferimento al Nazionale:

![richiesta trasferimento il progetto gaia croce rossa italiana](https://cloud.githubusercontent.com/assets/3315913/19396587/1a793898-9244-11e6-8daa-b5d470b0c8d9.png)

Cosa che posso tranquillamente fare. Se il mio presidente lo approva o passano 30 giorni divento socio del Comitato Nazionale. Questo penso che apra grossi problemi, ad esempio sul tesseramento.

Credo che sia una cosa molto urgente da prendere in considerazione... @AlfioEmanueleFresta @PaoloGiustiniani @matthewmazzoleni @galamarco

**Comment by PaoloGiustiniani** [comment]
@luca-dex puoi fare review?

**Comment by matthewmazzoleni** [comment]
arrivano diverse segnalazioni riguardanti trasferimenti negati dal presidente uscente ma che dopo 30gg sono state comunque automaticamente approvate dal sistema.
